modules/nf-core/sgdemux/main.nf

```diff
@@ -41,4 +40,31 @@ process SGDEMUX {
         sgdemux: \$(echo \$(sgdemux --version 2>&1) | cut -d " " -f2)
     END_VERSIONS
     """
+
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    mkdir -p ${prefix}
+
+    for sample in \$(cut -d, -f2 ${sample_sheet} | tail -n +2 ); do
+        uppercase_sample=\$(echo \$sample | tr '[:lower:]' '[:upper:]')
+        echo | gzip > ${prefix}/\${sample}_\${uppercase_sample}_L001_R1_001.fastq.gz
+        echo | gzip > ${prefix}/\${sample}_\${uppercase_sample}_L001_R2_001.fastq.gz
+    done
```

Comment:
This seems a little excessive, you could just make some files called e.g. `test1` etc.